cfgov/jinja2/v1/_layouts/base.html

```diff
@@ -124,22 +124,24 @@
 <a href="#main" id="skip-nav">Skip to main content</a>
 
 {% if flag_enabled('BETA_NOTICE') %}
-<div class="beta-banner expandable" id="beta-banner">
+<div class="beta-banner m-expandable m-expandable__expanded" id="beta-banner">
     <div class="wrapper wrapper__match-content">
         <div class="beta-banner_head">
             <span class="cf-icon cf-icon-error-round beta-banner_icon"></span>
             This beta site is a work in progress.
         </div>
-        <p class="beta-banner_desc expandable_content">
-            We’re prototyping new designs. Things may not work as expected.
-            Our regular site continues to be at
-            <a href="http://www.consumerfinance.gov/">www.consumerfinance.gov</a>.
-        </p>
-        <button class="btn beta-banner_btn expandable_target" id="beta-banner_btn">
-            <span class="expandable_cue-close">
+        <div class="m-expandable_content">
+            <p class="beta-banner_desc  m-expandable_content-animated">
+                We’re prototyping new designs. Things may not work as expected.
+                Our regular site continues to be at
+                <a href="http://www.consumerfinance.gov/">www.consumerfinance.gov</a>.
+            </p>
+        </div>
+        <button class="btn beta-banner_btn m-expandable_target m-expandable_link" id="beta-banner_btn">
```

Comment:
More than three classnames should be stacked. Double check these files elsewhere.